sdk/storage/storage-file/test/aborter.test.ts

```diff
@@ -2,21 +2,30 @@ import * as assert from "assert";
 
 import { Aborter } from "../src/Aborter";
 import { ShareURL } from "../src/ShareURL";
-import { getBSU, getUniqueName } from "./utils";
+import { getBSU } from "./utils";
+import { record } from "./utils/nock-recorder";
 import * as dotenv from "dotenv";
-dotenv.config({path:"../.env"});
+dotenv.config({ path:"../.env" });
 
 // tslint:disable:no-empty
-describe("Aborter", () => {
+describe("Aborter", function() {
   const serviceURL = getBSU();
-  let shareName: string = getUniqueName("share");
-  let shareURL = ShareURL.fromServiceURL(serviceURL, shareName);
+  let shareName: string;
+  let shareURL: ShareURL;
+  const testSuiteTitle = this.fullTitle();
+
+  let recorder: any;
 
   beforeEach(async () => {
-    shareName = getUniqueName("share");
+    recorder = record(testSuiteTitle, this.ctx.currentTest!.title);
+    shareName = recorder.getUniqueName("share");
```

Comment:
This line of code
  > shareName = getUniqueName("share");

is moved from describe to beforeeach!
Considering the offline discussion about recording_before file, should this be moved to before instead ?

Reply:
Actually this line of code was present both in 'describe' and in 'beforeEach', but its value in 'describe' was not being used because it was being overwritten in 'beforeEach'. I believe that moving it to 'before' would not break any test, but I'm trying to keep the changes to the tests to a minimum.

Reply:
Makes sense, this is alright.
I didn't see `shareName = getUniqueName("share");` in the beforeeach.